Dockerfile.compute-node

```diff
@@ -754,16 +773,27 @@ RUN rm /usr/local/pgsql/lib/lib*.a
 # Extenstion only
 #
 #########################################################################################
+FROM python:3.11-slim-bookworm AS generate-ext-index
+ARG PG_VERSION
+ARG BUILD_TAG
+# Define extension build numbers
+# NOTE: it is *not* necessary for the build tag to be BUILD_TAG. In particular,
```

Comment:
Until we don't have a clear way how to do it, let's pass BUILD_TAG as a command-line argument and not through a file

Reply:
Having this file right now makes it easier to enable de-duplication in the future.

Reply:
ok, we will remove it for now